src/signals/incident/components/form/ContainerSelect/Selector/Selector.tsx

```diff
@@ -148,26 +159,73 @@ const Selector = () => {
 
   const mapWrapper =
     <Wrapper data-testid="containerSelectSelector">
-      <StyledMap hasZoomControls mapOptions={mapOptions} setInstance={setMap} events={{}}>
-        <ViewerContainer
-          topLeft={
-            <ButtonBar>
-              <div>
-                <Button onClick={addContainer}>Container toevoegen</Button>
-                <Button onClick={removeContainer}>Container verwijderen</Button>
-                <Button onClick={close}>Meld deze container/Sluiten</Button>
-              </div>
-              <Paragraph as="h6">
-                Geselecteerd: {selection ? `[${selection.reduce((res, { id }) => `${res},${id}`, '')}]` : '<geen>'}
-              </Paragraph>
-            </ButtonBar>
-          }
-        />
+      <StyledMap
+        data-testid="map"
+        hasZoomControls={showDesktopVariant}
+        mapOptions={mapOptions}
+        setInstance={setMap}
+        events={{}}
+      >
+        <MapPanelProvider
+          mapPanelSnapPositions={{
```

Comment:
Is this value a constant? Could it be then in the beginning of the file defined? This would make the passed props more readable.